presto-redshift/pom.xml

```diff
@@ -16,12 +16,25 @@
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
     </properties>
 
+    <repositories>
+        <repository>
+            <id>redshift</id>
+            <url>http://redshift-maven-repository.s3-website-us-east-1.amazonaws.com/release</url>
```

Comment:
please disable snapshots for this repository  as in https://github.com/trinodb/trino/pull/6486/commits/7960ab1c6ca2adc5b1025b506456e8ae5ad93b29